chia/wallet/wallet_node.py

```diff
@@ -747,6 +763,14 @@ async def disconnect_and_stop_wpeers(self):
             await self.wallet_peers.ensure_is_closed()
             self.wallet_peers = None
 
+    async def check_for_synced_peer(self, header_block: HeaderBlock, request_time: uint64):
```

Comment:
Isn't this `check_for_synced_trusted_peer()`?

```suggestion
    async def check_for_synced_peer(self, header_block: HeaderBlock, request_time: uint64) -> bool:
```